.clang-tidy

```diff
@@ -1,6 +1,6 @@
 ---
 Checks:          '-*,clang-analyzer-core.*,clang-diagnostic-*'
-WarningsAsErrors: 'clang-analyzer-unix.*,clang-analyzer-core.NullDereference'
+WarningsAsErrors: 'clang-analyzer-unix.*,clang-analyzer-core.NullDereference,-clang-diagnostic-implicit-fallthrough'
```

Comment:
Why are you turning off the implicit fallthough check? It is a quite common error and can be hard to find.

Reply:
It's not turning it off, this is just disabling it being turned into an error. Currently we have too many fall throughs like this without a good way to handle it in a portable way.